src/app/shared/example-viewer/example-viewer.scss

```diff
@@ -21,23 +25,31 @@
   flex: 1 1 auto;
 }
 
-.docs-example-source-copy {
-  position: absolute;
-  top: 8px;
-  display: none;
+.docs-example-viewer-body {
+  padding: 30px;
 }
 
-.docs-example-source-wrapper:hover {
-  .docs-example-source-copy {
-    display: inline-block;
-  }
+.button-bar {
+  float: right;
+}
+
+code-snippet {
+  padding: 20px;
+  background: rgba(0, 0, 0, 0.01);
 }
 
 .docs-example-source {
-  padding: 0 30px 10px 30px;
-  min-height: 150px;
+  padding: 0;
+  margin: 0;
+  border: none;
+  background: none;
 }
 
-.docs-example-viewer-body {
-  padding: 30px;
+.docs-example-source-copy {
+  right: 0;
```

Comment:
IIRC something was setting this to like `9px` or something, can we find that and change it rather than overriding?